drivers/ukvirtio/blk/Config.uk

```diff
@@ -0,0 +1,9 @@
+config VIRTIO_BLK
+	bool "Virtio Block Device"
+	default y if LIBUKBLKDEV
```

Comment:
This can go away, as we can't tell with certainty with drivers we should select if `blkdev` is enabled. This logic belongs to the platform configuration.